back/engines/commercial/content_builder/app/models/content_builder/patches/project.rb

```diff
@@ -36,6 +36,10 @@ module Project
         pg_search_scope :search_by_content_layouts, associated_against: {
           content_builder_layouts: [Arel.sql(CRAFTJS_TEXT_QUERY)]
         }, using: { tsearch: { prefix: true } }
+
+        def uses_content_builder?
+          content_builder_layouts.present? && content_builder_layouts.pluck(:enabled).include?(true)
```

Comment:
I think the whole line could be replaced by `content_builder_layouts.any?(&:enabled)`.